<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalendarApp</class>
 <widget class="QMainWindow" name="CalendarApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CalendarApp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>461</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(239, 246, 255);
selection-color: rgb(3, 3, 4);
selection-background-color: rgb(237, 234, 239);
alternate-background-color: rgb(241, 238, 244);
gridline-color: rgb(174, 172, 176);
font: 13pt &quot;American Typewriter&quot;;</string>
    </property>
    <property name="firstDayOfWeek">
     <enum>Qt::Sunday</enum>
    </property>
    <property name="gridVisible">
     <bool>true</bool>
    </property>
    <property name="horizontalHeaderFormat">
     <enum>QCalendarWidget::SingleLetterDayNames</enum>
    </property>
    <property name="verticalHeaderFormat">
     <enum>QCalendarWidget::NoVerticalHeader</enum>
    </property>
    <property name="navigationBarVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="bookPushButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>550</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">selection-color: rgb(0, 0, 0);
selection-background-color: rgb(237, 234, 239);
font: 13pt &quot;American Typewriter&quot;;</string>
    </property>
    <property name="text">
     <string>Book</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;American Typewriter&quot;;</string>
    </property>
    <property name="text">
     <string>Plan your visit with Dr. Goodman</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="feedbackLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>550</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;American Typewriter&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>80</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;American Typewriter&quot;;</string>
    </property>
    <property name="text">
     <string>Step 1: 
Find a date</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>120</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;American Typewriter&quot;;</string>
    </property>
    <property name="text">
     <string>Step 2: 
Click on the date 
to see the availablity</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>190</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;American Typewriter&quot;;</string>
    </property>
    <property name="text">
     <string>Step 3: 
Click on the desired 
time slot, click &quot;Book&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>260</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;American Typewriter&quot;;</string>
    </property>
    <property name="text">
     <string>Step 4: 
Fill in name, email,
and reason for visit
in popped out form</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>350</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;American Typewriter&quot;;</string>
    </property>
    <property name="text">
     <string>Step 5: 
Click on &quot;Send&quot;. 
Success or Failed will 
show besides &quot;Book&quot; 
indicating if your 
request has been sent</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>480</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;American Typewriter&quot;;</string>
    </property>
    <property name="text">
     <string>All set! A staff will 
contact you by email </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>40</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;American Typewriter&quot;;</string>
    </property>
    <property name="text">
     <string>Instructions:</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>0</y>
      <width>811</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 200, 220);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>400</y>
       <width>461</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(239, 246, 255);
selection-color: rgb(0, 0, 0);
selection-background-color: rgb(237, 234, 239);
font: 13pt &quot;American Typewriter&quot;;</string>
     </property>
    </widget>
   </widget>
   <zorder>frame</zorder>
   <zorder>calendarWidget</zorder>
   <zorder>bookPushButton</zorder>
   <zorder>label</zorder>
   <zorder>feedbackLabel</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
